### PR TITLE
[Docs] Update docs for ext_oneapi_queue_priority on HIP

### PR DESCRIPTION
**Pull request by al42and**
https://github.com/oneapi-src/unified-runtime/pull/1229

### CODE REVIEW

File: sycl/plugins/unified_runtime/CMakeLists.txt [hunk 1]
```diff
@@ -50,14 +50,8 @@ endif()
...
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  #commit b38855ed815ffd076bfde5e5e06170ca4f723dc1
-  #Merge: e6343f4 6a2c548
-  #Author: Piotr Balcer <piotr.balcer@intel.com>
-  #Date:   Thu Oct 5 12:15:42 2023 +0200
-  #    Merge pull request #920 from jsji/localcopy
-  #        [UR][L0] Copy prebuilt L0 to avoid leaking shared folder path
-  set(UNIFIED_RUNTIME_TAG b38855ed815ffd076bfde5e5e06170ca4f723dc1)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/al42and/unified-runtime.git")
+  set(UNIFIED_RUNTIME_TAG "ext_oneapi_queue_priority-hip")
```
Comment (smaslov-intel) [inline sycl/plugins/unified_runtime/CMakeLists.txt:54]: why is this not merged to [regular ](https://github.com/oneapi-src/unified-runtime.git)?

Comment (al42and) [reply sycl/plugins/unified_runtime/CMakeLists.txt:54]: This is a draft PR per the contribution guide of unified-runtime, for https://github.com/oneapi-src/unified-runtime/pull/984

Comment (smaslov-intel) [reply sycl/plugins/unified_runtime/CMakeLists.txt:54]: ok, so it's for testing it only not to merge

**Review by smaslov-intel — APPROVED** [review]

File: sycl/plugins/unified_runtime/CMakeLists.txt (hunk 1 above)
Comment (al42and) [reply sycl/plugins/unified_runtime/CMakeLists.txt:54]: Yes. Should I have indicated it differently?

Comment (smaslov-intel) [reply sycl/plugins/unified_runtime/CMakeLists.txt:54]: Perhaps add this info in the description and link to the UR PR

**Review by kbenzie — APPROVED** [review]

**Comment by kbenzie** [comment]
Going to have to revert the UR change and put this back to draft until we fix the build failure.

**Comment by kbenzie** [comment]
@intel/llvm-gatekeepers please merge

**Pull request merged**